meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx6ullevk-sec/close.uuu

```diff
@@ -0,0 +1,23 @@
+uuu_version 1.2.39
+
+SDP: boot -f SPL-mfgtool.signed -dcdaddr 0x0907000 -cleardcd
+
+SDPV: delay 1000
+SDPV: write -f u-boot-mfgtool.itb
+SDPV: jump
+
+FB: ucmd if mmc dev 0; then setenv fiohab_dev 0; else setenv fiohab_dev 1; fi;
+
+FB: ucmd setenv srk_0 0xEA2F0B50
+FB: ucmd setenv srk_1 0x871167F7
+FB: ucmd setenv srk_2 0xF5CECF5D
+FB: ucmd setenv srk_3 0x364727C3
+FB: ucmd setenv srk_4 0x8DD52832
+FB: ucmd setenv srk_5 0xF158F65F
+FB: ucmd setenv srk_6 0xA71BBE78
+FB: ucmd setenv srk_7 0xA3AD024A
+
+FB: ucmd if fiohab_close; then echo Platform Secured; else echo Error, Can Not Secure the Platform; fi
```

Comment:
This might need update based on https://github.com/foundriesio/lmp-tools/pull/12, so make sure to be aligned with that pr.